convert2rhel/data/8/x86_64/configs/centos-8-x86_64.cfg

```diff
@@ -1,9 +1,9 @@
 [system_info]
 
-# Fingerprints of GPG keys used for signing packages of the OS to be converted.
+# key_ids of GPG keys used for signing packages of the OS to be converted.
```

Comment:
I think with these we can say more normally what key_ids is

Reply:
will change, thanks!